README.md

```diff
@@ -123,8 +144,11 @@ I'm not aware of any, if you are please [make a pull request](http://makeapullre
-| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+
+| [<img src="https://avatars.githubusercontent.com/u/1500684?v=3" width="100px;"/><br /><sub>Kent C. Dodds</sub>](https://kentcdodds.com)<br />[💻](https://github.com/kentcdodds/rtl-css-js/commits?author=kentcdodds 'Code') [⚠️](https://github.com/kentcdodds/rtl-css-js/commits?author=kentcdodds 'Tests') [🚇](#infra-kentcdodds 'Infrastructure (Hosting, Build-Tools, etc)') | [<img src="https://avatars.githubusercontent.com/u/63876?v=3" width="100px;"/><br /><sub>Ahmed El Gabri</sub>](https://gabri.me)<br />[💻](https://github.com/kentcdodds/rtl-css-js/commits?author=ahmedelgabri 'Code') [📖](https://github.com/kentcdodds/rtl-css-js/commits?author=ahmedelgabri 'Documentation') [⚠️](https://github.com/kentcdodds/rtl-css-js/commits?author=ahmedelgabri 'Tests') | [<img src="https://avatars1.githubusercontent.com/u/1383861?v=4" width="100px;"/><br /><sub>Maja Wichrowska</sub>](https://github.com/majapw)<br />[💻](https://github.com/kentcdodds/rtl-css-js/commits?author=majapw 'Code') [⚠️](https://github.com/kentcdodds/rtl-css-js/commits?author=majapw 'Tests') | [<img src="https://avatars2.githubusercontent.com/u/6600720?v=4" width="100px;"/><br /><sub>Yaniv</sub>](https://github.com/yzimet)<br />[💻](https://github.com/kentcdodds/rtl-css-js/commits?author=yzimet 'Code') [⚠️](https://github.com/kentcdodds/rtl-css-js/commits?author=yzimet 'Tests') | [<img src="https://avatars2.githubusercontent.com/u/5658514?v=4" width="100px;"/><br /><sub>Jonathan Pollak</sub>](https://github.com/TxHawks)<br />[💻](https://github.com/kentcdodds/rtl-css-js/commits?author=TxHawks 'Code') [⚠️](https://github.com/kentcdodds/rtl-css-js/commits?author=TxHawks 'Tests') | [<img src="https://avatars1.githubusercontent.com/u/8528759?v=4" width="100px;"/><br /><sub>Ali Taheri Moghaddar</sub>](https://github.com/alitaheri)<br />[💻](https://github.com/kentcdodds/rtl-css-js/commits?author=alitaheri 'Code') [📖](https://github.com/kentcdodds/rtl-css-js/commits?author=alitaheri 'Documentation') [⚠️](https://github.com/kentcdodds/rtl-css-js/commits?author=alitaheri 'Tests') | [<img src="https://avatars0.githubusercontent.com/u/844459?v=4" width="100px;"/><br /><sub>garrettberg</sub>](https://github.com/garrettberg)<br />[💻](https://github.com/kentcdodds/rtl-css-js/commits?author=garrettberg 'Code') [⚠️](https://github.com/kentcdodds/rtl-css-js/commits?author=garrettberg 'Tests') |
```

Comment:
this change probably done by prettier

Reply:
Hmm... That's weird. all-contributors-cli was updated to add a `prettier-ignore` comment above this. Try running `npx kcd-scripts contributors generate` and it should fix this and add a `prettier-ignore` comment automatically.